components/eamxx/cime_config/testdefs/testmods_dirs/scream/mam4xx/drydep/shell_commands

```diff
@@ -0,0 +1,13 @@
+
+#Default scream has 10 tracers, MAM4xx adds another 31 making a total of 41 tracer
+#Set total number of tracers to 41. We are using append here as last entry wins while parsing xml options
+./xmlchange --append SCREAM_CMAKE_OPTIONS="SCREAM_NUM_TRACERS 41"
```

Comment:
I left a similar comment on @singhbalwinder 's ACI PR:
Here we assume a default of 10 tracers in EAMxx, but as the model evolves we may expect some configurations that could have >10.  @singhbalwinder did you put something in your PR that queried the number of tracers before adding 31?  Maybe there is no easy solution other than what you have done here by including a comment.

Reply:
Actually, this is for a test, so maybe this is sufficient since we can assume that the test is setup to use 10 tracers for SCREAM.